docs/how_to/query.md

```diff
@@ -51,6 +51,25 @@ You will notice that in the setup linked above, M3DB has just one unaggregated n
     all: false
 ```
 
+## ID generation
+
+The default generation scheme for IDs is unfortunately prone to collisions, but is left as is for back-compat reasons. It is suggested to set the ID generation scheme to one of either "quoted" or "prepend_meta". Quoted generation scheme yields the most human-readable IDs, whereas Prepend_Meta is better for more compact IDS, or if tags are expected to contain non-ASCII characters. To set the ID generation scheme, add the following to your coordinator configuration yaml file:
+
+```yaml
+tagOptions:
+  idScheme: <name>
+```
+
+As an example of how these schemes generate tags, consider a case with 4 tags, [{t1:v1}, {t2:v2}, {t3:v3}, {t4:v4}]. The following is an example of how different schemes will generate IDs.
+
+nil: t1=v1,t2=v2,t3=v3,t4=v4,
+prepend_meta: 2,2,2,2,2,2,2,2!t1v1t2v2t3v3t4v4
+quoted: {t1="v1",t2="v2",t3="v3",t4="v4"}
```

Comment:
what do you do if tag name/value includes quote characters themselves? i.e. is there escaping too?

Reply:
Yeah, good call will call out escaping specifically